PlayBook/src/keymapping/using_making_keymaps.md

```diff
@@ -1,59 +1,42 @@
 # Using & Making Keymaps
 
-Many apps and games already have full keymaps available on the Discord. For more info see [this article](./import_export_keymaps.md).
+Many apps and games already have full keymaps available on the Discord. For more info see [this article](./import_export_keymaps.md). Additionally, you can  find them on the [keymaps repository](https://github.com/PlayCover/keymaps). 
```

Comment:
The GitHub repo is the main source to be linked from the docs, as it is more up to date and there is more documentation about them.